docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+
+services:
+  localstack:
+    image: localstack/localstack:1.3.0
+    ports:
+      - "127.0.0.1:5001:5001"
+    volumes:
+      - "./src/main/resources/localstack/init-aws.sh:/etc/localstack/init/ready.d/init-aws.sh"
```

Comment:
these init-aws.sh didn't run on my windows docker setup it seemed


localstack_1  | 2022-12-19T05:29:10.050 ERROR --- [  MainThread] localstack.runtime.init    : Error while running script Script(path='/etc/localstack/init/ready.d/init-aws.sh', stage=READY, state=ERROR): [Errno 2] No such file or directory: '/etc/localstack/init/ready.d/init-aws.sh'

so i need to run these manually
aws s3 --endpoint-url http://localhost:5001 mb s3://test-core-bucket
aws s3 --endpoint-url http://localhost:5001 cp .\src\main\resources\localstack\s3\  s3://test-core-bucket/ --recursive

Reply:
This is baffling 😓 @gmsdelmundo-ttd had the same issue as you, we were about to debug it today. Works on both my laptop and desktop flawlessly though. Will  take a further look with you/Gian as I can't recreate it.

Reply:
ok, should be fixed, seems to have been due to this issue: https://github.com/tianon/docker-bash/issues/9. Thanks for the help @gmsdelmundo-ttd !